.git_hooks/pre-commit

```diff
@@ -17,3 +17,7 @@ if [ -n "${INTERNAL_URLS}" ]; then
   echo >&2 "${INTERNAL_URLS}"
   exit 1
 fi
+
+if [ "$COMMIT_CHECK" == "true" ]; then
```

Comment:
I have the discipline to run that manually

Reply:
ok, so, you don't need this
i forget it on regular basis